src/frontend/planner_test/tests/testdata/bushy_join.yaml

```diff
@@ -604,35 +567,6 @@
     order by
       nation,
       o_year desc;
-  stream_plan: |
```

Comment:
Does the streaming plan miss?

Reply:
I rerun the apply-planner-test, and find
```
<   stream_error: |-
<     Not supported: streaming nested-loop join
<     HINT: The non-equal join in the query requires a nested-loop join executor, which could be very expensive to run. Consider rewriting the query to use dynamic filter as a substitute if possible.
<     See also: https://github.com/risingwavelabs/rfcs/blob/main/rfcs/0033-dynamic-filter.md
```